lib/main.ts

```diff
@@ -92,13 +97,16 @@ function symbolCompareFunction(a: ts.Symbol, b: ts.Symbol) {
 }
 
 /**
- * Traverses the node tree to construct the text without comments.
+ * Traverses the node tree to construct the text without comments and privates.
  */
-function getTextWithoutComments(node: ts.Node): string {
-  // TODO: test more
+function getSanitizedText(node: ts.Node): string {
+  if (node.flags & ts.NodeFlags.Private) {
```

Comment:
if you mean `constructor(private a: string)`, I believe it is always converted to
```
private a;
constructor(a: string)
```
by the d.ts generator.